scenarios/command/dotnet/command_client/Worker.cs

```diff
@@ -32,8 +32,8 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             while (!stoppingToken.IsCancellationRequested)
             {
                 _logger.LogInformation("Invoking Command: {time}", DateTimeOffset.Now);
-                unlockResponse response = await commandClient.InvokeAsync("vehicle03",
-                    new unlockRequest
+                UnlockResponse response = await commandClient.InvokeAsync("vehicle03",
+                    new UnlockRequest
                     {
                         When = DateTime.Now.ToUniversalTime().ToTimestamp(),
```

Comment:
I don't know where `ToTimestamp()` comes from, but would this work?

```suggestion
                        When = DateTime.UtcNow.ToTimestamp(),
```

Reply:
https://learn.microsoft.com/en-us/aspnet/core/grpc/protobuf?view=aspnetcore-7.0#dates-and-times

Reply:
oh I see.. sure